ledger/sandbox-classic/src/test/suite/scala/platform/sandbox/DevModeIT.scala

```diff
@@ -0,0 +1,75 @@
+package com.daml.platform.sandbox
+
+import java.nio.file.{Files, Path, Paths}
+
+import com.daml.bazeltools.BazelRunfiles.rlocation
+import com.daml.grpc.adapter.{ExecutionSequencerFactory, SingleThreadExecutionSequencerPool}
+import com.daml.ledger.api.refinements.ApiTypes.ApplicationId
+import com.daml.ledger.client.LedgerClient
+import com.daml.ledger.client.configuration.{
+  CommandClientConfiguration,
+  LedgerClientConfiguration,
+  LedgerIdRequirement
+}
+import com.daml.platform.apiserver.services.GrpcClientResource
+import com.daml.platform.sandbox.config.SandboxConfig
+import com.daml.resources.ResourceOwner
+import com.google.protobuf.ByteString
+import org.scalatest.{Matchers, WordSpec}
+
+import scala.concurrent.Await
+import scala.concurrent.ExecutionContext.Implicits._
+import scala.concurrent.duration._
+
+class DevModeIT extends WordSpec with Matchers {
+
+  private[this] implicit val esf: ExecutionSequencerFactory =
+    new SingleThreadExecutionSequencerPool("testSequencerPool")
+
+  val List(stableDarPath, devDarPath) =
+    List("1.8", "1.dev").map(s => Paths.get(rlocation(s"daml-lf/encoder/test-$s.dar")))
+
+  private[this] val applicationId = ApplicationId("DevModeIT")
+
+  private[this] def ledgerClientConfiguration =
+    LedgerClientConfiguration(
+      applicationId = ApplicationId.unwrap(applicationId),
+      ledgerIdRequirement = LedgerIdRequirement.none,
+      commandClient = CommandClientConfiguration.default,
+      sslContext = None,
+      token = None
+    )
+
+  private[this] def sandboxSever(devMode: Boolean) =
+    SandboxServer.owner(SandboxConfig.defaultConfig.copy(devMode = devMode))
```

Comment:
You can use a random port:

```suggestion
    SandboxServer.owner(SandboxConfig.defaultConfig.copy(
      devMode = devMode,
      port = Port.Dynamic,
    ))
```

Reply:
Cool.